### ISSUE TITLE
Configurable product

### ISSUE THREAD

**Issue by simplehumble** [bigbridge-nl/product-import#8]
Could you give working example for a-configurable.xml in example folder? I can add the product, and its visible only in admin area. Not visible in front page. I am suspecting example xml file is missing some attributes. Could you please take a look into this?

**Comment by patrick-bigbridge** [comment]
Ah yes, a-configurable.xml only shows the configurable itself, and only the sku's of the simples it links to. The simples themselves need to be added too, and they need a "color" attribute.

I will update the example tonight.

**Comment by garfix** [comment]
Hi, I extended a-configurable.xml with all the properties needed for a real-life example.

https://github.com/bigbridge-nl/product-import/blob/master/doc/example/a-configurable.xml

I used the following command to import it; it auto creates attribute options for "color" and auto creates categories:

`bin/magento bigbridge:product:import --auto-create-option=color --auto-create-categories vendor/bigbridge/product-import/doc/example/a-configurable.xml
`

I struggled with this task as well. I thought the attribute "is_in_stock" would not be necessary for the configurable (only for the simples below it), but apparently it is.
